tests/tests.py

```diff
@@ -454,13 +454,61 @@ def test_res_comp_with_no_podpolicies(self):
                 removed = True
             else:
                 time.sleep(1)
+    
+    def test_appstatus_plugin(self):
 
+        def cleanup():
+            cmd = "cd ./hello-world"
+            TestKubePlus.run_command(cmd)
+            cmd = "kubectl delete -f hs1.yaml --kubeconfig=provider.conf"
+            TestKubePlus.run_command(cmd)
+            cmd = "kubectl delete -f hello-world-service-composition.yaml --kubeconfig=provider.conf"
+
+        if not TestKubePlus._is_kubeplus_running():
+            print("KubePlus is not running. Deploy KubePlus and then run tests")
+            sys.exit(0)
+
+        if os.getenv("KUBEPLUS_HOME") == '':
+            print("Skipping test as KUBEPLUS_HOME is not set.")
+            return
+        
+        # add Kubeplus provider
+        cmd = "cp ../kubeplus-saas-provider.json ./hello-world/provider.conf"
+        TestKubePlus.run_command(cmd)
+
+        # register HelloWorldService API
+        cmd = "kubectl create -f ./hello-world/hello-world-service-composition.yaml --kubeconfig=./hello-world/provider.conf"
+        TestKubePlus.run_command(cmd)
+
+        # check CRD installation
+        crd = "helloworldservices.platformapi.kubeplus"
+        crd_installed = self._check_crd_installed(crd)
+        if not crd_installed:
+            print("CRD " + crd + " not installed. Exiting this test.")
+            return
+        
+        # create app instance
+        cmd = "kubectl create -f ./hello-world/hs1.yaml --kubeconfig=./hello-world/provider.conf"
+        out, err = TestKubePlus.run_command(cmd)
+
+        # test plugin
+        cmd = "kubectl appstatus HelloWorldService hs1"
```

Comment:
Don't we need to pass "-k provider.conf" here?

Reply:
While I was testing locally, it worked without `-k provider.conf` (as does appresources), but I will add it in.